### PR TITLE
refactor: Move contributors rendering logic to separate file "Contributors.astro"

### PR DESCRIPTION
**Pull request by RaulCatalinas**

### CODE REVIEW

File: src/layouts/ContributorLayout.astro
Comment (polacodev) [inline src/layouts/ContributorLayout.astro:1]: this must be a component, Layouts are Astro Components used to provide a reusable UI structure, such as a page template.
take a look here https://docs.astro.build/en/basics/layouts/

Comment (RaulCatalinas) [reply src/layouts/ContributorLayout.astro:1]: I have already corrected it

Comment (polacodev) [reply src/layouts/ContributorLayout.astro:1]: great, you rock!

File: src/components/Contributors/ContributorLayout.astro
Comment (polacodev) [inline src/components/Contributors/ContributorLayout.astro:1]: change the filename plese, this is not a layout
e.g ContributorContainer

File: package.json
Comment (polacodev) [inline package.json:1]: hey @RaulCatalinas I could not get why are you adding toast messages, 

can you show me how is the preview with this changes, 
you can add screenshots for more context btw, or feel free to reach to me

Comment (RaulCatalinas) [reply package.json:1]: The toast messages are only displayed in browsers that do not support the ShareAPI (e.g. Firefox) and are to notify the user whether the operation went well or badly.

This is the one that is displayed when all is well

image

If you don't like the toast messages I can change the styles or remove them directly.

Comment (polacodev) [reply package.json:1]: looks great, but the main purpose of the application is to provide libraries, I think we are going in the wrong direction, let's remove for now

Comment (RaulCatalinas) [reply package.json:1]: Done, I've deleted it.
I just wanted to give users an easy way to share the website.

File: src/components/Contributors/ContributorLayout.astro
Comment (polacodev) [reply src/components/Contributors/ContributorLayout.astro:1]: @RaulCatalinas you have here another one, thie has to be  with a component name and not a layout

**Pull request merged**